airbyte_cdk/test/entrypoint_wrapper.py

```diff
@@ -157,6 +157,8 @@ def _run_command(
     stream_handler.setFormatter(AirbyteLogFormatter())
     parent_logger = logging.getLogger("")
     parent_logger.addHandler(stream_handler)
+    if "--debug" not in args:
```

Comment:
Not needed for tests to pass. Just enabling more logs in the tests for debugging purposes